core/src/main/java/io/questdb/cutlass/text/TextImportRequestJob.java

```diff
@@ -69,22 +78,23 @@ public TextImportRequestJob(
     ) throws SqlException {
         this.requestQueue = engine.getMessageBus().getTextImportRequestQueue();
         this.requestSubSeq = engine.getMessageBus().getTextImportRequestSubSeq();
-        this.importer = new ParallelCsvFileImporter(engine, workerCount);
+        this.parallelImporter = new ParallelCsvFileImporter(engine, workerCount);
+        this.serialImporter = new SerialCsvFileImporter(engine);
 
         CairoConfiguration configuration = engine.getConfiguration();
         this.clock = configuration.getMicrosecondClock();
-        this.statusTableName = configuration.getSystemTableNamePrefix() + "parallel_text_import_log";
+        this.statusTableName = configuration.getSystemTableNamePrefix() + "text_import_log";
 
         this.sqlCompiler = new SqlCompiler(engine, functionFactoryCache, null);
         this.sqlExecutionContext = new SqlExecutionContextImpl(engine, 1);
         this.sqlExecutionContext.with(AllowAllCairoSecurityContext.INSTANCE, null, null);
         this.sqlCompiler.compile(
                 "CREATE TABLE IF NOT EXISTS \"" + statusTableName + "\" (" +
                         "ts timestamp, " + // 0
-                        "id symbol, " + // 1
+                        "id string, " + // 1
```

Comment:
nit: symbol is slightly more efficient in terms of the on-disk footprint, so I'd keep it unless there is a reason to use a string.

Reply:
Nope, there is no compaction, but on the other hand the symbol table would be growing quite slowly, so it's not a huge concern.